mungegithub/github/github.go

```diff
@@ -932,7 +956,12 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, error) {
 	if !obj.IsPR() {
 		return nil, fmt.Errorf("Issue: %d is not a PR", *obj.Issue.Number)
 	}
-	return obj.RefreshPR()
+	pr, err := obj.config.getPR(*obj.Issue.Number)
```

Comment:
Yes. So the next caller to GetPR() won't hit the Github API server again. Only an explicit call to Refresh() should cause us to hit it again.